community/learn/graphql-tutorials/tutorials/typescript-react-apollo/tutorial-site/content/mutations-variables/1-create-todo.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Create todos - mutation"
+metaTitle: "Mutation to create todos | GraphQL React Apollo Typescript Tutorial"
+metaDescription: "GraphQL Mutation to create new personal todos. Try the mutation in GraphiQL, passing the Authorization token to get authenticated results."
+---
+
+In this part of the tutorial, you will learn how to create new todos by using GraphQL Mutations.
+
+Let's define a graphql mutation to perform insert into todos.
+
+```graphql
+mutation insert_todos($todo: String!, $isPublic: Boolean!) {
+  insert_todos(objects: {title: $todo, is_public: $isPublic}) {
+    affected_rows
+    returning {
+      id
+      title
+      created_at
+      is_completed
+    }
+  }
+}
+```
+
+You will also need to pass in the values for the variables.
+
+[Try](https://learn.hasura.io/graphql/graphiql) this mutation in GraphiQL against the application database to see what the response looks like.
```

Comment:
I couldn't see the hyperlink and was gonna ask you to hyperlink.
Good chance that people wouldn't see it either.
If possible, hyperlink the phrase `Try this mutation in GraphiQL`.

Reply:
Similar for rest of the places where  only `Try`  has been hyperlinked.